Dockerfile

```diff
@@ -20,10 +23,12 @@ RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/$NVM_VERSION/insta
     && npm install -g npm@$NPM_VERSION yarn@$YARN_VERSION
 
 RUN apt-get update -y \
-    && apt-get install -y shared-mime-info=1.5-2ubuntu0.2\
+    && apt-get install -y shared-mime-info=1.15-1\
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+RUN mv /etc/apt/sources.list.d.bak /etc/apt/sources.list.d
```

Comment:
Should do this all inline before installing packages, like power_jobs.

Reply:
You're right. I moved it because there was a different error, which I incorrectly thought was related to the line being where it was.